src/enums.rs

```diff
@@ -311,25 +309,76 @@ impl EditCommand {
             | EditCommand::CutRightUntil(_)
             | EditCommand::CutRightBefore(_)
             | EditCommand::CutLeftUntil(_)
-            | EditCommand::CutLeftBefore(_) => UndoBehavior::Full,
+            | EditCommand::CutLeftBefore(_) => EditType::EditText,
 
-            EditCommand::Undo | EditCommand::Redo => UndoBehavior::Ignore,
+            EditCommand::Undo | EditCommand::Redo => EditType::UndoRedo,
         }
     }
 }
 
-/// Specifies how the (previously executed) operation should be treated in the Undo stack.
+/// Specifies the types of edit commands, used to simplify grouping edits
+/// to mark undo behavior
+#[derive(PartialEq)]
```

Comment:
Clippy got a bit more pedantic with todays release: 

```suggestion
#[derive(PartialEq, Eq)]
```